packages/components/src/components/drawer/drawer.custom.styles.ts

```diff
@@ -67,7 +67,7 @@ export default css`
    */
   .drawer__title,
   .header-actions {
```

Comment:
I just noticed, that we have a wrong className here. We don't have any `header-actions` class in the drawer component. Therefore the `min-height` is never added somewhere. Should be `drawer__header-actions` instead

 ```suggestion
  .drawer__header-actions {
```

Reply:
Fixed && thanks :)